QueueManagerTest/QueueManagerTests.cs

```diff
@@ -1,20 +1,46 @@
 ﻿using System.Collections.Generic;
 using NUnit.Framework;
 using QueueManager;
+using System.IO;
+using System;  
 
 namespace QueueManagerTest
 {
     [TestFixture]
     public class QueueManagerTests
     {
         QueueManagers _queueManager;
+        string _pathString;
+        string _upperLevelFolderName;
+        string _testFileName;
+        TextWriter _testFile;
+        string _pathWithFile;
+        List<string> _linesInFile;
 
         [SetUp]
         public void SetUp()
         {
             _queueManager = new QueueManagers();
+            _upperLevelFolderName = @"..\..\Test Files\";
+            _pathString = Path.GetFullPath(_upperLevelFolderName);
```

Comment:
Ah, I see your point.  You're right.  I guess we should remove this.  I'm sorry.  I misunderstood what you meant before.